src/meander/environment/epsilon.cljc

```diff
@@ -35,8 +35,27 @@
    :meander.epsilon/rewrite-vector-as-to-and true
    :meander.epsilon/substitute-acyclic-references true
    ;; When set to true disables type checks and bounds checks.
+   :meander.epsilon/use-native-methods true
    :meander.epsilon/unsafe false
```

Comment:
We can remove these two fields because their meaning is given by the fields they desugar to.